qiskit/providers/ibmq/api_v2/rest/job.py

```diff
@@ -127,7 +130,7 @@ def get_object_storage(self, url):
         return self.session.get(url, bare=True).json()
 
 
-def build_url_filter(excluded_fields, included_fields):
+def build_url_filter(excluded_fields: List[str], included_fields: List[str]) -> Dict[str, Any]:
```

Comment:
Can you check if it is possible to be more specific about the return type? This function has bitten us in a number of issues and is a bit special compared to the rest of functions in the module (as in, it does not rely on a generic dict return by the api, but something we generate), and actually a good example where typing can be useful.